src/Plugin/media/Source/GoogleForm.php

```diff
@@ -26,6 +31,31 @@ class GoogleForm extends MediaSourceBase implements MediaSourceFieldConstraintsI
    */
   const METADATA_ATTRIBUTE_ID = 'id';
 
+  /**
+   * The name of the height field.
+   *
+   * @var string
+   */
+  protected $heightField;
```

Comment:
This property appears to be unused. Lets remove it.

Reply:
So, I put this in to support the `getHeightField` function originally.  per: https://github.com/SU-SWS/stanford_media/pull/81#discussion_r533667359

Would you prefer I keep the property an implement the `getHeightField` function, or remove the property and rely on configuration in the `GoogleFormFormatter`?

Reply:
Either way results in the same outcome. Whichever you'd like to go.

Reply:
This is now in use.